modules/Bio/EnsEMBL/Compara/PipeConfig/HomologyAnnotation_conf.pm

```diff
@@ -126,9 +135,9 @@ sub core_pipeline_analyses {
     );
 
     my %blastp_parameters = (
-        'blast_bin_dir'         => $self->o('blast_bin_dir'),
-        'blast_params'          => "#expr( #all_blast_params#->[#param_index#]->[2])expr#",
-        'evalue_limit'          => "#expr( #all_blast_params#->[#param_index#]->[3])expr#",
+        'diamond_exe'           => $self->o('diamond_exe'),
+        'blast_params'          => $self->o('blast_params'),
+        'evalue_limit'          => $self->o('evalue_limit'),
     );
```

Comment:
Actually it looks like `%blastp_parameters` is not used in this PipeConfig so maybe it could be removed ?